src/lib.rs

```diff
@@ -98,7 +98,7 @@ pub trait Cpu {
 
     fn emu(&self) -> &Unicorn;
```

Comment:
No need, only need the box when we want to use the `Hookable` trait implemented on `Box<Unicorn>`